apps/files/models.py

```diff
@@ -131,10 +131,15 @@ def get_mirror(self, addon, attachment=False):
         return posixpath.join(*map(smart_str, [host, addon.id, self.filename]))
 
     def get_url_path(self, src):
+        return self._make_download_url('downloads.file', src)
```

Comment:
Since this is probably used all over and removing it would suck maybe it's worth adding a dosctring suggesting that `get_absolute_url` be used instead?